src/NullCache.php

```diff
@@ -33,7 +35,7 @@ public function get($key, $default = null)
 
     public function getMultiple($keys, $default = null): iterable
     {
-        return array_fill_keys($keys, $default);
+        return array_fill_keys((array)$keys, $default);
```

Comment:
Keys should always be array. Not sure we need such type-cast.

Reply:
Interface says it's iterable

Reply:
Ah, correct. Are you sure type-casting works well? There's https://www.php.net/manual/en/function.iterator-to-array.php

Reply:
Yes, casting for an array works correctly for all iterables except `IteratorAggregate` and generators.
I think we need to use this:
```php
$array = $iterable instanceof \Traversable ? iterator_to_array($iterable) : $iterable;
```

Reply:
Or just `iterator_to_array` all the time.

Reply:
It will fail with array `iterator_to_array([])` it only accepts Traversable